lib/cpp/src/thrift/transport/TSSLSocket.cpp

```diff
@@ -576,12 +576,12 @@ int TSSLSocketFactory::passwordCallback(char* password, int size, int, void* dat
   TSSLSocketFactory* factory = (TSSLSocketFactory*)data;
   string userPassword;
   factory->getPassword(userPassword, size);
-  int length = userPassword.size();
+  std::string::size_type length = userPassword.size();
   if (length > size) {
```

Comment:
std::string::size_time is the same as the return value of std::string::size(), which is potentially different than "int" (on Windows this was a compiler warning), which is why I changed it to quiet the warnings.  It is more correct now.